src/backend/joanie/core/utils/course_run.py

```diff
@@ -0,0 +1,39 @@
+"""
+Utility for course runs on the amount of active enrollments once the course run has reached its
+end date, and the amount of orders to unlock the accessibility to get the certificate.
+"""
+import logging
+
+from django.core.exceptions import ValidationError
+from django.utils import timezone as django_timezone
+
+from joanie.core import enums
+from joanie.core.models import CourseRun, Enrollment, Order
+
+logger = logging.getLogger(__name__)
+
+
+def enrollments_by_resource_link(resource_link: str):
+    """
+    Returns a dictionary containing the following values from a `resource_link` of a course run:
+        - amount of enrollments at the end date of a course run,
+        - amount of orders made to unlock the certificates at the end of a course run.
+    """
+    if not CourseRun.objects.filter(resource_link=resource_link).exists():
+        error_message = "Make sure to give an existing resource link from a course run."
+        logger.error("Error: %s", error_message)
+        raise ValidationError(error_message)
```

Comment:
As you must return those counters when the course run is ended, I'll check first if the course run is ended then I'll return counters if there is. But this logic, trigger me another question. Let's talk on Slack with @NamFra about that.

```suggestion
    try:
    	course_run = CourseRun.objects.get(resource_link=resource_link, end__lte=timezone.now())
    except CourseRun.DoesNotExist as exception:
        error_message = "Make sure to give an existing resource link from an ended course run."
        logger.error("Error: %s", error_message)
        raise ValidationError(error_message) from exception
```

Reply:
Perfect ! You suggestion is valid, let's go.